x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/planner/QueryTranslatorTests.java

```diff
@@ -2697,8 +2697,49 @@ public void testSubqueryGroupByFilterAndOrderByByRealiased() {
         optimizeAndPlan("SELECT g as h FROM (SELECT date AS f, int AS g FROM test) WHERE h IS NOT NULL GROUP BY h ORDER BY h ASC");
     }
     
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/69758")
-    public void testFilterAfterGroupBy() {
-        optimizeAndPlan("SELECT j AS k FROM (SELECT i AS j FROM ( SELECT int AS i FROM test) GROUP BY j) WHERE j < 5");
+    public void testFilterOnAggregatesSameAsHaving() {
+        EsQueryExec havingPlan =
```

Comment:
For this and the other tests, it would be nice to also have a simple query (without subselect) and check that the plans are equivalent.

Reply:
Not sure what do you mean by "simple query". Can you give me an example?

Reply:
`SELECT bool as b, AVG(int) as a FROM test GROUP BY bool HAVING AVG(int) > 2`